packages/xsnap/package.json

```diff
@@ -4,18 +4,16 @@
   "description": "Description forthcoming.",
   "author": "Agoric",
   "license": "Apache-2.0",
-  "parsers": {
-    "js": "mjs"
-  },
+  "type": "module",
   "main": "./src/index.js",
   "bin": {
     "ava-xs": "./src/ava-xs.js",
     "xsrepl": "./src/xsrepl"
   },
   "scripts": {
-    "repl": "node -r esm src/xsrepl.js",
+    "repl": "node src/xsrepl.js",
     "build:bundle": "rollup --config rollup.config.js",
-    "build:bin": "node -r esm src/build.js",
+    "build:bin": "node src/build.js",
```

Comment:
This seems to trigger the build failure:

```
$ yarn build:bin && yarn build:bundle
$ node src/build.js
ReferenceError: __dirname is not defined
    at file:///home/runner/work/agoric-sdk/agoric-sdk/packages/xsnap/src/build.js:47:33
```
https://github.com/Agoric/agoric-sdk/pull/3564/checks?check_run_id=3198495455#step:7:158